### PR TITLE
Add Open Source Initiative Icon

### PR DESCRIPTION
**Pull request by luispuerto**
**Issue:** #1551

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- The hex is the body of the logo.
- I download the logo from the website.
- I just used the official website.

### CODE REVIEW

**Comment by luispuerto** [comment]
Addresses #1551

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2145,6 +2145,11 @@
             "hex": "F56C2D",
             "source": "https://www.origin.com/gbr/en-us/store"
         },
+        {
+            "title": "OSI",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: I would say the title should be "Open Source Initiative", any particular reason you choose "OSI" instead?

```suggestion
            "title": "Open Source Initiative",
```

**Comment by luispuerto** [comment]
I think I've fixed 🙂

**Comment by ericcornelissen** [comment]
Thanks for your work on this @luispuerto :tada: Again, it is very much appreciated!

**Pull request merged**